docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -600,6 +602,8 @@ This event will be sent when final composed recording is generated and uploaded
         "room_name": "TestRoom1",
         "session_id": "************************",
```

Comment:
also, in sample event let's remove URL field and just show recording_presigned_url.

Reply:
Removed deprecated fields everywhere.